tests/test_printers.py

```diff
@@ -42,7 +42,8 @@ def test_find_dict_composite_key_value_fails():
     ('foo', 'yaml', printers.YamlPrinter),
 ])
 def test_get_asset_printer(asset, output_format, printer_cls):
-    assert isinstance(printers.get_asset_printer(asset, output_format), printer_cls)
+    def get_asset_printer(ctx):  # need to use this method to pass the ctx parameter
+        assert isinstance(printers.get_asset_printer(asset, ctx.obj.output_format), printer_cls)
```

Comment:
There is an issue here: you're defining a function that will do the assert but never trigger the function. So the test passes but only because there is actual assertion.

That being said, since `get_asset_printer` isn't modified at all, you should be able to just remove all changes from this file.

Reply:
It works, indeed. I change that in the next commit! 👍